support/testutil/fake.go

```diff
@@ -30,3 +30,7 @@ func (f *fakeImageProvider) GetImage(key string) string {
 func (f *fakeImageProvider) ImageExist(key string) (string, bool) {
 	return key, true
 }
+
+func (f *fakeImageProvider) ComponentImages() map[string]string { // not used
```

Comment:
Is this still needed?

Reply:
Yes, it is.  'cause the ImageProvider interface now exposes `ComponentImages`. That explains also why I added the last commit (with that strange comment which will be squashed anyway).